docs/book/extensions.md

```diff
@@ -1,29 +1,57 @@
 
-Examples of such extensions are the [@litexa/gadgets extension](gadgets-custom-interfaces.html#installation), the
-[@litexa/apl extension](screens.html#installation), and the
-[@litexa/render-template extension](appendix-render-template.html#installation). These extensions
-allow for optional customizations of Litexa, and can be installed on a project-by-project basis,
-depending on requirements.
+The Alexa Games team has written a set of extensions to add higher level support for some features
+we use. If you find yourself needing functionality absent in Litexa, or wanting to customise something
+to your specific workflow you may want to look into writing your own extension. Open an issue on
+the Litexa Github if you'd like to talk to us about your use case.
+
+
+## Extension Directory
+
+To discover which extensions, both local and global, are visible to your current skill, run the command
+`litexa extensions`. That will give you a printout of each extension's name and location on disk.
+
+Here's a list of public extensions currently available via npm:
+
+* [**litexa/deploy-aws:**](https://www.npmjs.com/package/@litexa/deploy-aws) A deployment module that pushes a skill to AWS using Lambda, DynamoDB, and S3.
+
+* [**litexa/html:**](https://www.npmjs.com/package/@litexa/html) An extension that simplifies launching and exchanging messages with a web app as part of the [Alexa Web API for Games](https://developer.amazon.com/en-US/blogs/alexa/alexa-skills-kit/2020/07/alexa-web-api-for-games-generally-available).
+
+* [**litexa/apl:**](https://www.npmjs.com/package/@litexa/apl) An extension that makes working with the Alexa Presentation Language (APL) in your Litexa project more powerful, with shorthand for managing APL documents and common design patterns.
+
+* [**litexa/render-template:**](https://www.npmjs.com/package/@litexa/render-template) An extension that supports easily building, sending, and validating a `Display.RenderTemplate` directive, the predecessor to APL.
+
+* [**litexa/assets-wav:**](https://www.npmjs.com/package/@litexa/assets-wav) A WAV/MP3 composer that can combine multiple overlapping samples into a single MP3 stream, and a binding layer for use in Literate Alexa.
+
+* [**litexa/gadgets:**](https://www.npmjs.com/package/@litexa/gadgets) An extension for the Gadgets Skill API, which powers interaction with Echo Buttons (and potentially other Alexa Gadgets).
+
+
+
+## Authoring new Extensions
 
 Litexa extensions support the following customizations:
 
 1. extending Litexa's language with new statements
 2. adding new validators to Litexa's compilation and testing steps
 3. adding new runtime functionality
 
+As each Litexa extension is its own node.js module, so step 1 in writing your own is to initialize a new module.
+During development, don't forget that npm supports installing a module by absolute and local references on your hard drive.
+Installing your extension into a test skill ASAP makes iterating much easier.
```

Comment:
I don't quite follow the relation between the 2 sentences.

Reply:
Came up during a conversation with someone: they didn't realize that you can "npm install" an arbitrary module on your HD by just folder reference, i.e. that you didn't have to give it a name and publish it to npm before you can test it. Installing a local module will let you apply it to a skill while you're developing it.

Reply:
Ah I see. I also missed that this was in the extensions page, that gave me more context.